app/views/layouts/public.html.haml

```diff
@@ -45,6 +45,6 @@
             %h4= t 'footer.more'
             %ul
               %li= link_to t('about.source_code'), Mastodon::Version.source_url
-              %li= link_to 'joinmastodon.org', 'https://joinmastodon.org'
+              %li= link_to t('about.apps'), 'https://joinmastodon.org/apps'
```

Comment:
Not sure about this one

Reply:
"What is mastodon?" links to joinmastodon.org so I figured there's no need to have duplicates.